docs/docs/08.1-more-about-refs.md

```diff
@@ -1,13 +1,97 @@
 ---
 id: more-about-refs
-title: More About Refs
+title: References to Components
```

Comment:
Can we call it "Refs to Components" here (and in the sidebar, if that's specified somewhere else)?